spec/presenters/finder_presenter_spec.rb

```diff
@@ -82,50 +82,85 @@
   end
 
   describe "#sort_options" do
+    def sort_option(label, value, disabled: false, selected: false)
+      disabled_attr = if disabled
```

Comment:
This may be cleaner:
```
disabled_attr = disabled ? 'disabled="disabled" ' : ''
```

Reply:
I think the linter will not like that, but maybe it's configured differently here. I'll give it a go.